modules/books.js

```diff
@@ -0,0 +1,92 @@
+import {titleInput, authorInput, bookList} from './values.js'
+
+let booksArray = [];
+
+export class BookEntry {
+  constructor(title, author, idNum) {
+    this.title = title;
+    this.author = author;
+    this.idNum = idNum;
+  }
+
+  add() {
+    const creaateAndSet = (title, author, idNum) => {
+      const bookBlock = document.createElement("div");
+      const bookInfo = document.createElement("div");
+      const bookTitle = document.createElement("h4");
+      const bookAuthor = document.createElement("h4");
+      const bookIdNumber = document.createElement("h4");
+      const removeButton = document.createElement("button");
+
+      bookTitle.innerHTML = `'${title}'`;
+      bookAuthor.innerHTML = `by ${author}`;
+      bookIdNumber.innerHTML = idNum;
+      removeButton.innerHTML = "Remove";
+      bookInfo.classList.add("bookinfo");
+      bookIdNumber.classList.add("counter");
+      removeButton.classList.add("btn", "rmbtn");
+
+      if (idNum % 2 === 0) {
+        bookBlock.classList.add("bookblock2");
+      } else {
+        bookBlock.classList.add("bookblock1");
+      }
+
+      bookInfo.append(bookTitle, bookAuthor, bookIdNumber);
+      bookBlock.append(bookInfo, removeButton);
+      bookList.append(bookBlock);
+
+      BookEntry.remove(removeButton);
+    };
+    creaateAndSet(this.title, this.author, this.idNum);
+  }
+
+  static remove(element) {
+    function removeBook() {
+      booksArray = booksArray.filter((book) => +book.idNum 
+      !== +this.parentNode.children[0].children[2].innerHTML);
+      this.parentNode.remove();
+
+      booksArray.forEach((book, i) => {
+        bookList.children[i].children[0].children[2].innerHTML = i;
```

Comment:
- [ ] **Required:** This is working as it should. Well done. However, you are required to favour the use of ES6 arrow functions where possible.

![image](https://user-images.githubusercontent.com/27709832/165194642-2f5c847e-c90f-4888-85b1-ea39857e601d.png)

Reply:
It was really complicated and I had to change quite a few things but I made it work and learned a lot, thanks!